spec/function.dd

```diff
@@ -2054,14 +2054,12 @@ void func(T)(T x)
 {
     writeln(x);
 }
-
```

Comment:
I don't see a pressing need. TDPL is full of such adjustments and they don't impede quality.